### ISSUE TITLE
Improve dropdown for Filters desktop_filter_style_2.html

### ISSUE THREAD

**Issue by Tushar98644** [Clueless-Community/seamless-ui#1492]
Need to improve this component on `\<Component name>\src\<File Name.html>`

## Improvements required :
dropdown not working 

## Any reference image?
<img width="1344" alt="Screenshot 2023-01-04 at 5 07 16 PM" src="https://user-images.githubusercontent.com/107763774/210547960-aed09997-9e1b-47ac-93b0-520701e9aed3.png">
  

## Any external reference?
  
You can checkout [this]() one for better understanding.

**Comment by RavindraP04** [comment]
It is working for me, Please Check again
![Screenshot_20230104_173209](https://user-images.githubusercontent.com/98400348/210551138-141c6cef-22b2-46ba-9ddb-e2b2c98b143a.png)
![Screenshot_20230104_173222](https://user-images.githubusercontent.com/98400348/210551144-5f8ab51f-b0e5-4960-964e-70fb5a664e65.png)

**Comment by Tushar98644** [comment]
in my case it's working but the click event is defined on the text and not on the dropdown button....that's why i missed it the first time

**Comment by RavindraP04** [comment]
Yes the actual working dropdown button is below the fixed svg but it was not aligning as required so if you can re align and resize it then it would work on clicking on button too 
![image](https://user-images.githubusercontent.com/98400348/210553874-8f8dcf05-3315-4acb-92a4-6ab0aa8761f8.png)

**Comment by Tushar98644** [comment]
> Yes the actual working dropdown button is below the fixed svg but it was not aligning as required so if you can re align and resize it then it would work on clicking on dropdown ison too ![image](https://user-images.githubusercontent.com/98400348/210553874-8f8dcf05-3315-4acb-92a4-6ab0aa8761f8.png)

ok......can i start working on this issue ?

**Comment by RavindraP04** [comment]
Yes you can, Make it work and just make the PR, in general If the issue is genuine don't wait for it to be assigned to you

**Comment by AtharvSabde** [comment]
can you please assign this issue

**Comment by RavindraP04** [comment]
> can you please assign this issue



Sure, Go ahead @AtharvSabde

**Comment by RavindraP04** [comment]
@AtharvSabde Any Update ?

**Comment by ManasaKaza** [comment]
Thank you

**Comment by Aditya-Raghuvanshi** [comment]
i want to work on this issue under JWOC.

**Comment by Aditya-Raghuvanshi** [comment]
can i start working on it ?
